### PR TITLE
Add changes to use in minicart

### PR DESCRIPTION
**Pull request by ArthurSampaio**
#### What is the purpose of this pull request?

Add changes to use in minicart.

#### What problem is this solving?

Adding delete button, borders and aligning component items

#### How should this be manually tested?
https://arthur--storecomponents.myvtex.com

#### Screenshots or example usage
pr-product-summary-01

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/index.js [hunk 1]
```diff
@@ -213,11 +227,13 @@ class ProductSummary extends Component {
         onMouseEnter={this.handleMouseEnter}
         onMouseLeave={this.handleMouseLeave}
       >
-        <div className="pointer pa2">
-          <Link
+        <div className={elementClasses}>
+
```
Comment (vitoriaHeliane) [inline react/index.js]: Remove the white line.

File: react/index.js [hunk 2]
```diff
@@ -244,8 +260,11 @@ class ProductSummary extends Component {
                 />
               </div>
             </div>
+
```
Comment (vitoriaHeliane) [inline react/index.js]: Same here

Comment (ArthurSampaio) [reply react/index.js]: done

File: react/index.js (hunk 1 above)
Comment (ArthurSampaio) [reply react/index.js]: done.

File: react/index.js [hunk 3]
```diff
@@ -204,10 +213,15 @@ class ProductSummary extends Component {
...
     })
 
     const informationClasses = classNames('vtex-product-summary__informations', {
-      'w-50 pv3 ph4': displayMode === 'inline',
+      'w-80 pv3 pl3 pr3 h-100': displayMode === 'inline'
+    })
+
+    const elementClasses = classNames('pointer pa2', {
+      'bb b--light-gray ma2': showBorders
```
Comment (theomoura) [inline react/index.js]: use design tokens

File: react/index.js [hunk 4]
```diff
@@ -60,6 +62,8 @@ class ProductSummary extends Component {
       'small',
       'inline',
     ]),
+    /** Define a function that is executed when a product is clicked */
+    actionOnClick: PropTypes.func,
```
Comment (gugabribeiro) [inline react/index.js]: "Function that is executed..." is enough :pick:

File: react/index.js [hunk 5]
```diff
@@ -77,6 +81,7 @@ class ProductSummary extends Component {
       showSku: false,
     },
     displayMode: 'normal',
+    showBorders: false
```
Comment (gugabribeiro) [inline react/index.js]: Add a comma at the end of this line :pick:

File: react/index.js [hunk 6]
```diff
@@ -204,10 +213,15 @@ class ProductSummary extends Component {
     const imageContainerClasses = classNames('vtex-product-summary__image-container db', {
       'w-100 center': displayMode !== 'inline',
       'w-40': displayMode === 'inline',
+
```
Comment (gugabribeiro) [inline react/index.js]: Remove this empty line :pick:

File: react/index.js [hunk 7]
```diff
@@ -250,7 +265,11 @@ class ProductSummary extends Component {
             </div>
           </Link>
           <div className={buyButtonClasses}>
+<<<<<<< HEAD
```
Comment (gugabribeiro) [inline react/index.js]: The merge operation was not concluded properly. 
Please fix asap.

**Pull request merged**